### PR TITLE
mavlink socket: stream to localhost before actually receiving packets

### PR DESCRIPTION
**Pull request by tumbili**

### CODE REVIEW

File: src/modules/mavlink/mavlink_main.h [hunk 1]
```diff
@@ -334,6 +335,8 @@ class Mavlink : public device::VDev
 
 	int 			get_socket_fd () { return _socket_fd; };
 
+	struct sockaddr_in * get_socket_source_address() {return &_src_addr;};
```
Comment (tumbili) [inline src/modules/mavlink/mavlink_main.h]: This will probably fail for NuttX since _src_addr doesn't exist

**Comment by tumbili** [comment]
@LorenzMeier Please comment.

File: src/modules/mavlink/mavlink_main.h (hunk 1 above)
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_main.h]: Travis CI delivers a concise build log. If you had already the hunch, why not push a commit that ifdefs this out?

**Comment by LorenzMeier** [comment]
Nice! Applied!

**Pull request closed without merging**